format/spec.md

```diff
@@ -1286,6 +1291,7 @@ This serialization scheme is for storing single values as individual binary valu
 | **`struct`**                 | Not supported                                                                                                |
 | **`list`**                   | Not supported                                                                                                |
 | **`map`**                    | Not supported                                                                                                |
+| **`geometry`**               | A single point, encoded as a {x, y, optional z, optional m} concatenation of its 8-byte IEEE 754 values, little-endian. |
```

Comment:
Is it always concatenated by 4 floating-point values? If it is not the case, we'll have a hard time figuring out if the point is in XYZ or XYM when there are 3 encoded dimensions. I suggest we use the WKB encoding of points here as well.

Reply:
+1. This is why we have introduced a separate bounding box stats in the Parquet proposal to avoid the issue.

Reply:
Thanks @Kontinuation for the suggestion.  Added specification to use NaN for a coordinate if unset.